src/autocompletePicker.ts

```diff
@@ -0,0 +1,224 @@
+/* Create and edit data on Organizations
+**
+** organizations conveys many distinct typed of thing.
+**
+*/
+import { icons, ns, style, widgets, store } from 'solid-ui'
+import { NamedNode, Store, st } from 'rdflib'
+import { queryPublicDataByName, filterByLanguage, wikidataParameters,
+  AUTOCOMPLETE_LIMIT, QueryParameters, getPreferredLanguages } from './publicData'
+
+const kb = store
+
+const AUTOCOMPLETE_THRESHOLD = 4 // don't check until this many characters typed
+const AUTOCOMPLETE_ROWS = 12 // 20?
+
+const autocompleteRowStyle = 'border: 0.2em solid straw;'
+
+/*
+Autocomplete hapopens in four phases:
+  - The saerch string is too small to bother
+  - The search string is big enough, and we have not loaded the arrray
+  - The search string is big enough, and we have loaded arrray up to the limit
+      Display them and wait for more user input
+  - The search string is big enough, and we have loaded arrray NOT to the limit
+     so in that case we have got all matching ones.   No more fetches
+     if user gets more precise, wait for them to select one - or reduce to a single
+  - Optionally waiting for accept button to be pressed
```

Comment:
Might be 5 phases (not 4) given the optional last line
```suggestion
Autocomplete happens in four phases:
  1. The search string is too small to bother
  2. The search string is big enough, and we have not loaded the array
  3. The search string is big enough, and we have loaded array up to the limit
       Display them and wait for more user input
  4. The search string is big enough, and we have loaded array NOT to the limit
     but including all matches.   No more fetches.
     If user gets more precise, wait for them to select one - or reduce to a single
  5. Optionally waiting for accept button to be pressed
```